tests/e2e/performance/test_pvc_bulk_creation_deletion_performance.py

```diff
@@ -79,28 +94,62 @@ def test_multiple_pvc_creation_measurement_performance(self, teardown_factory):
                 )
 
                 executor.submit(pvc_obj.reload)
+
         start_time = helpers.get_provision_time(
             self.interface, pvc_objs, status="start"
         )
         end_time = helpers.get_provision_time(self.interface, pvc_objs, status="end")
-        total = end_time - start_time
-        total_time = total.total_seconds()
-        logging.info(f"{number_of_pvcs} PVCs creation time is {total_time} seconds.")
+        total_time = (end_time - start_time).total_seconds()
+        logging.info(f"{bulk_size} Bulk PVCs creation time is {total_time} seconds.")
 
-        if total_time > 60:
+        if total_time > bulk_creation_time_limit:
             raise ex.PerformanceException(
-                f"{number_of_pvcs} PVCs creation time is {total_time} and "
-                f"greater than {multi_creation_time_limit} seconds"
+                f"{bulk_size} Bulk PVCs creation time is {total_time} and "
+                f"greater than {bulk_creation_time_limit} seconds"
             )
-        logging.info(f"{number_of_pvcs} PVCs creation time took {total_time} seconds")
 
-    @pytest.mark.usefixtures(base_setup.__name__)
+        pv_names_list = []
+        for pvc_obj in pvc_objs:
+            pv_names_list.append(pvc_obj.backed_pv)
+
+        logging.info(f"Starting to delete bulk of {bulk_size} PVCs")
+        helpers.delete_bulk_pvcs(yaml_creation_dir, pv_names_list)
+        logging.info(f"Deletion of bulk of {bulk_size} PVCs successfully completed")
+
+        start_deletion_time = helpers.get_pvc_bulk_deletion_time(
+            self.interface, pv_names_list, status="start"
+        )
+        end_deletion_time = helpers.get_pvc_bulk_deletion_time(
+            self.interface, pv_names_list, status="end"
+        )
+
+        total_deletion_time = (end_deletion_time - start_deletion_time).total_seconds()
+        logging.info(
+            f"{bulk_size} Bulk PVCs deletion time is {total_deletion_time} seconds."
+        )
+
+    @pytest.fixture()
+    def base_setup_creation_after_deletion(
+        self, interface_iterate, storageclass_factory
+    ):
+        """
+        A setup phase for the test
+
+        Args:
+            interface_iterate: A fixture to iterate over ceph interfaces
+            storageclass_factory: A fixture to create everything needed for a
+                storageclass
+        """
+        self.interface = interface_iterate
+        self.sc_obj = storageclass_factory(self.interface)
+
+    @pytest.mark.usefixtures(base_setup_creation_after_deletion.__name__)
     @polarion_id("OCS-1270")
     @bugzilla("1741612")
-    def test_multiple_pvc_creation_after_deletion_performance(self, teardown_factory):
+    def test_bulk_pvc_creation_after_deletion_performance(self, teardown_factory):
```

Comment:
It's better to move this function to performance helper file than using in testcase

Reply:
@ramkiperiy I am afraid I do not understand. 
test_bulk_pvc_creation_after_deletion_performance  is a test case and not a function, we need it as a test.

Reply:
ah ok!! sorry my bad.